movie-browser/src/App.js

```diff
@@ -28,9 +29,9 @@ toggleMovieSection = () => {
     return (
       <div>
         <Header />
-        <Featured movies={movies} />
-        <MostViewed movies={movies} />
-        <Movies movies={movies} toggleMovieSection={this.toggleMovieSection} shouldShowAllMovies={this.state.shouldShowAllMovies} />
+        <Featured featuredMovies={movies} />
+        <MostViewed mostViewedMovies={movies} />
```

Comment:
same thing in here, u only need 6, not all of them

Reply:
Same splice idea as I did with `Featured`?

Reply:
yes